controllers/fenceagentsremediation_controller_test.go

```diff
@@ -148,9 +148,12 @@ var _ = Describe("FAR Controller", func() {
 		When("creating valid FAR CR", func() {
 			BeforeEach(func() {
 				node = utils.GetNode("", workerNode)
+				underTestFAR = getFenceAgentsRemediation(workerNode, fenceAgentIPMI, testShareParam, testNodeParam)
 			})
 			It("should have finalizer, taint, while the two VAs and one pod will be deleted", func() {
 				By("Searching for remediation taint")
+				nodeKey.Name = workerNode
```

Comment:
Modifying variables on the fly is confusing when reading the test
Why not using directly the `client.ObjectKey`?

```
client.ObjectKey{Name: workerNode, Namespace: defaultNamespace}
```